risc0/zkvm/src/host/server/prove/tests.rs

```diff
@@ -87,20 +73,20 @@ fn hashfn_poseidon2() {
     prove_nothing("poseidon2").unwrap();
 }
 
-#[test]
-fn hashfn_blake2b() {
-    let hal_pair = HalPair {
-        hal: Rc::new(CpuHal::new(Blake2bCpuHashSuite::new_suite())),
-        circuit_hal: Rc::new(CpuCircuitHal::new()),
-    };
-    let env = ExecutorEnv::builder()
-        .write(&MultiTestSpec::DoNothing)
-        .unwrap()
-        .build()
-        .unwrap();
-    let prover = ProverImpl::new("cpu:blake2b", hal_pair, ReceiptKind::Composite);
-    prover.prove(env, MULTI_TEST_ELF).unwrap();
-}
+// #[test]
```

Comment:
Should we add an issue to track this getting re-enabled?

Reply:
I plan to address this soon. I'm trying to get this landed so we can do an rc, then I'll go right back to fixing all the loose ends.